src/managed/Microsoft.Extensions.DependencyModel/DependencyContextJsonReader.Utf8JsonReader.cs

```diff
@@ -3,13 +3,16 @@
 
 using System;
 using System.Buffers;
+using System.Diagnostics;
 using System.IO;
 using System.Text.Json;
 
 namespace Microsoft.Extensions.DependencyModel
 {
     public partial class DependencyContextJsonReader : IDependencyContextReader
     {
+        private static ReadOnlySpan<byte> Utf8Bom => new byte[] { 0xEF, 0xBB, 0xBF };
```

Comment:
Would this be better as a method? That way someone wouldn't make the mistake of calling it more than once in a method.

Reply:
Perhaps.  The only pattern I've seen for a ReadOnlySpan over the program `.text` segment used a property; but the JIT or crossgen should see a property get just as a normal method... so it should work?

But for now I'm going with "this looks like it does in corefx" and there is "it looks like everywhere else".  But I'll definitely think on it going forward.